ui-frontend/packages/catalog-ui-search/src/main/webapp/component/query-editor/query-editor.view.js

```diff
@@ -154,9 +159,13 @@ module.exports = Marionette.LayoutView.extend({
     }
     queryContentView.save()
     this.queryTitle.currentView.save()
+    if (this.model.get('title') === '') {
```

Comment:
✏️ Since you're using it in multiple places why not pull it out into a function. Something like:

```
setDefaultTitleIfExists() {
  if (this.model.get('title') === '')
    this.setDefaultTitle()
}
```
then you can just call that function `this.setDefaultTitleIfExists()`